packages/workgrid-courier/package.json

```
@@ -10,7 +10,8 @@
   ],
   "scripts": {
     "clean": "rm -rf dist docs temp",
-    "build": "yarn clean && tsc && webpack",
+    "build": "yarn clean && webpack",
+    "postbuild": "tsc --emitDeclarationOnly"
```

Bah, forgot the comma!
```suggestion
    "postbuild": "tsc --emitDeclarationOnly",
```